setup.cfg

```
@@ -37,3 +37,10 @@ default_section=FIRSTPARTY
 known_first_party=inmanta,inmanta_ext,inmanta_plugins
 known_third_party=pytest,psutil,pydantic,pkg_resources
 skip=tests/data
+
+[mypy]
+plugins = pydantic.mypy
+
+[pydantic-mypy]
+init_forbid_extra=True
+init_typed=True
```

This is only a suggestion. What's your opinion on adding this option? It seems like it would provide a stronger guarantee.
```suggestion
init_typed=True
warn_untyped_fields=True
```